ee/hogai/graph/session_summaries/nodes.py

```diff
@@ -87,6 +88,18 @@ async def _stream_notebook_content(self, content: dict, state: AssistantState, p
         # Stream the notebook update
         await self._write_message(notebook_message)
 
+    def _has_video_validation_feature_flag(self) -> bool:
+        """
+        Check if the user has the video validation for session summaries feature flag enabled.
+        """
+        return posthoganalytics.feature_enabled(
+            "max-session-summarization-video-validation",
+            str(self._user.distinct_id),
+            groups={"organization": str(self._team.organization_id)},
+            group_properties={"organization": {"id": str(self._team.organization_id)}},
+            send_feature_flag_events=False,
+        )
```

Comment:
This is duplicated from `ee/api/session_summaries.py`, does it make sense too make this a shared function?

Reply:
It can be done, but it's a single function call without any specific logic (just checking a flag), so not sure if it's worth creating as an util, linking to two files in different places, and so.